python/lsst/afw/geom/ellipses/PixelRegion.i

```diff
@@ -59,12 +59,15 @@
 
 %extend PyPixelRegionIterator {
 %pythoncode %{
-def next(self):
+def __next__(self):
     if self.atEnd():
         raise StopIteration()
     current = self.get()
     self.increment()
     return current
+
+next = __next__
```

Comment:
I had not spotted that inheriting from `object` automatically added the old `next` method as a clone for `__next__`.